Plugin.pm

```diff
@@ -1274,7 +1282,6 @@ sub _trackItem {
 		}
 		$item->{displayWork} = $item->{work};
 		$item->{displayWork} = $track->{composer}->{name} . string('COLON') . ' ' . $item->{work} if ($track->{composer}->{name});
-		$item->{work} = $track->{composer}->{name} . string('COLON') . ' ' . $item->{work} if ($track->{composer}->{name});
```

Comment:
Is this no longer required? I don't see it set elsewhere?

Reply:
No, not in that format. The composer name is still prepended to `work` to create `displayWork`, which is used for display purposes. However, the tracks are now grouped by `work` only, resulting in a format that is identical to that shown in the Qobuz app. It gets tricky when the first track of a work is non-streamable though (meaning no composer). The composer name for the work must then be picked up from the first streamable track in the work, if any. I have seen some works consisting of just one non-streamable track and, for them, the `displayWork` field is the same as `work`. I'm hoping to get some feedback from Darrell on this change too but my testing looks very good thus far.

Reply:
_maybe it would be better for this change to be described as something like "ensure tracks in the same work are grouped even if composer metadata is missing from some"._

Good point. I agree.